src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/data-governance.mdx

```diff
@@ -7,7 +7,151 @@ tags:
 metaDescription: How to manage your data from the Kubernetes integration.
 freshnessValidatedDate: never
 ---
+## Logs
+The collection of  [Logs](https://github.com/newrelic/helm-charts/blob/master/charts/newrelic-logging/values.yaml#L27) in Kubernetes is  done  using the [New Relic kubernetes plugin](https://docs.newrelic.com/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/), an output plugin for Fluentbit.
 
+The configuration of **fluentbit** includes:
+- **Input Plugin**: This is where we define the log sources that we wish to collect, as in our case we are talking about k8s logs.
+- **Parser Plugins**: to convert and structure log messages (JSON, Regexp, LTSV, Logfmt, etc.)
+- **Filter Plugins**: to modify, enrich and delete   information from your logs (Nest, Throttle, Expect, Geoplp, Grep, K8s, etc.) For filtering, you can also use a LUA script.
+- **Output Plugins**: send the log stream to several outputs which will be responsible for storing the log stream and displaying them. In this case, we speak of NRDB (New Relic Database).
+
+Logs can be filtered following the methods below:
+### Change the path
+In the input configuration, we have the **path** defined, which would point the fluentbit agent to the **log paths** that need to be collected.
+
+For k8s containers, logs are stored in files with the format: **"/var/log/containers/<podname>_<namespace>....log"** (file example: **/var/log/containers/nri-bundle-newrelic-logging-pjdfks_default_newrelic-logging-1d45d23135d7062c1fcd7c872472a74d5294c55edd0592a2130c10b046234.log**)
+
+If you would like to concentrate for example the collection of logs on 3, 4 pods: you would have to specify the paths according to the pods names and the wildcard  (*):
+
+```yaml
+fluentBit:
+  logLevel: "info"
+  path: "/var/log/containers/myPod1*.log, /var/log/containers/myPod2*.log"
+```
+
+So it ends up looking like:
```

Comment:
what ends up looking like? Can you unspool how the first fluentbi yaml file leads to this global config? What's the connection between the two?